docs/tutorials/tracking/cell_tracking.md

```diff
@@ -172,7 +172,7 @@ We can now visualize the full, linked tracks in napari!
 ```python
 viewer = napari.Viewer()
 viewer.add_image(timelapse, scale=SCALE, name='Fluo-N3DH-CE')
-viewer.add_tracks(data, properties=properties, graph=graph, scale=SCALE, name='tracks')
+viewer.add_tracks(data, features=features, graph=graph, scale=SCALE, name='tracks')
```

Comment:
I have not tried upstreaming these changes to https://github.com/quantumjot/btrack/blob/caa56fa82330e4b16b5d28150f9b60ed963165c7/examples/napari_btrack.ipynb

Given their simplicity, I'm fairly confident things should work, but it's probably worth verifying.